src/main/webapp/i18n/en/universityInfo.json

```diff
@@ -0,0 +1,10 @@
+{
+    "artemisApp": {
+        "universityInfo": {
+            "title": "University",
+            "name": "Name",
+            "adminName": "Main Admins Name",
```

Comment:
This deviates quite a lot from the German translation. Why did you not simply use `administrator` as translation?

Reply:
Probably that would be the better approach. I changed it to `Administrator`